ToDoApp/TextField/CustomButton .swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CustomTextField.swift
+//  ToDoApp
+//
+//  Created by deniss.lobacs on 25/02/2022.
+//
+
+import Foundation
+import UIKit
+
+
+ class CustomButton: UIButton {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    init(placeholder: String) {
```

Comment:
i think, that it's bad idea, to set placeHolder at init, let's create a new function
public func setupPlaceHolder(title: String)